src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -68,45 +68,6 @@ private SystemEvents()
             // This class is intended to be static, but predates static classes (which were introduced in C# 2.0).
         }
 
-        // stole from SystemInformation... if we get SystemInformation moved
-        // to somewhere that we can use it... rip this!
-        [SuppressMessage("Microsoft.Reliability", "CA2006:UseSafeHandleToEncapsulateNativeResources")]
-        private static volatile IntPtr s_processWinStation = IntPtr.Zero;
-        private static volatile bool s_isUserInteractive = false;
-        private static unsafe bool UserInteractive
-        {
-            get
-            {
-                if (Environment.OSVersion.Platform == System.PlatformID.Win32NT)
```

Comment:
You can delete this condition if you would like to. We are not supporting Win95 anymore.